test/azure/Expected/AcceptanceTests/AzureParameterGrouping/azureparametergrouping/aio/operations/_parameter_grouping_operations.py

```diff
@@ -47,9 +47,43 @@ def __init__(self, client, config, serializer, deserializer) -> None:
         self._deserialize = deserializer
         self._config = config
 
+    def _post_required_request(
+        self, path: str, body: int, custom_header: Optional[str] = None, query: Optional[int] = 30, **kwargs: Any
```

Comment:
for parameter grouping, I'm ungrouping in the high level operation, and not passing the grouped model to the request.

designed this way bc imo llc is about removing models etc, didn't seem right to have users construct a model to pass to the request.

additionally, in the high level operation, the ungrouped parameters are all private (i.e. `_path`). Making them public for the request.

question of "what if users have a param called `path`, and and the grouped model input has a property also called `path`. Then, we would need to do some name padding for the ungrouped `path` parameter. Need to look more into whether this beahvior is allowed for users as well